connector/spanmetricsconnector/config.go

```diff
@@ -58,6 +58,9 @@ type Config struct {
 
 	// Exemplars defines the configuration for exemplars.
 	Exemplars ExemplarsConfig `mapstructure:"exemplars"`
+
+	// Events defines the configuration for events section of spans.
+	Events Event `mapstructure:"events"`
```

Comment:
I suggest we rename the type to `EventsConfig` to be consistent with other config types, and also maintaining consistency with the plural form "events":

```suggestion
	Events EventsConfig `mapstructure:"events"`
```